src/myst.ts

```diff
@@ -125,4 +147,6 @@ export function parseContent(notebook: StaticNotebook): void {
     cell.myst.post = mdast.children[index];
     cell.mystRender();
   });
+
+  return Promise.all(promises).then(() => undefined);
```

Comment:
This content now waits on the render, and then returns so that execution can happen. We must parse the cells BEFORE we execute.